client/src/App.js

```diff
@@ -26,28 +26,25 @@ const App = () => {
 		<main role="main">
 			<Router>
 				<nav className="navbar">
-					<ul className="nav-element">
-						<li className="nav-links">
-							<Link to="/Students" exact="true">
-                Student
-							</Link>
-						</li>
-						<li className="nav-links">
-							<Link to="/Mentors" exact="true">
-                Mentor
-							</Link>
-						</li>
-						<li className="nav-links">
-							<Link to="/Results" exact="true">
-                Quiz rezults
-							</Link>
-						</li>
-						<li className="nav-links">
-							<Link to="/NewQuestion" exact="true">
-                New question
-							</Link>
-						</li>
-					</ul>
+					<Link to="/" exact="true">
+						<img
+							src="https://codeyourfuture.io/wp-content/uploads/2019/03/cyf_brand.png"
```

Comment:
Just did a project for Hackathon involving CYF logo usage if they are going to use that and this we can use some style from there i'll show you later to keep product line consistent